tests/components/skybell/test_config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Test SkyBell config flow."""
+from unittest.mock import patch
+
+from aioskybell import exceptions
+
+from homeassistant.components.skybell.const import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from . import CONF_CONFIG_FLOW, _patch_skybell
+
+from tests.common import MockConfigEntry
+
+
+def _patch_setup() -> None:
+    return patch(
+        "homeassistant.components.skybell.async_setup_entry",
+        return_value=True,
+    )
+
+
+async def test_flow_user(hass: HomeAssistant) -> None:
+    """Test that the user step works."""
+    with patch("aioskybell.UTILS"), _patch_skybell():
```

Comment:
Patch `aioskybell` where its imported

Reply:
`UTILS` is not used by config flow so I changed the imports.